src/scripts/tbtcv2-rewards/rewards.ts

```diff
@@ -372,12 +372,7 @@ export async function calculateRewards() {
           // upgrade cutoff date that happens to be right before the interval
           // end date. However, it might still be eligible for rewards because
           // of the uptime requirement.
-          if (
-            !(
-              instances[i].buildVersion.includes(latestClientTag) ||
-              instances[i].buildVersion.includes(secondToLatestClientTag)
-            )
-          ) {
+          if (!eligibleClientTags.includes(instances[i].buildVersion)) {
```

Comment:
If I'm understanding it correctly, in the case of three versions (tags), the same cutoff date is set for the first and the second version, right?

I mean, if we have a distribution where `-m4`, `-m5` and `-m6` versions have been valid, we are setting that -m4 and -m5 are valid until cutoff date. This is, we are not checking if the node ran `-m4` version after the three-week deadline (in which case, the node should be punished).

Is it intended to simplify the application of the rules?

Reply:
Correct, this is to simplify rules and not force stakers to update their nodes one after another in a short period of time.